apps/console/src/features/users/components/bulk-import-response-list.tsx

```diff
@@ -249,13 +301,65 @@ export const BulkImportResponseList: React.FunctionComponent<BulkImportResponseL
         );
     };
 
+    /**
+     * Handles the flow mode switch.
+     *
+     * @param _ - Event.
+     * @param newTabIndex - New tab index.
+     */
+    const handleTabChange = (_: React.SyntheticEvent, newTabIndex: number): void => {
+        bulkResponseOperationTypes.map((item: DropdownItemProps) => {
+            if (item.key === newTabIndex) {
+                setResponseOperationType(item.value as BulkImportResponseOperationTypes);
+                setResponseOperationTypeTab(newTabIndex);
+            }
+        });
```

Comment:
Use `find` instead of `map` to avoid unnecessary iterations.

```
const item = bulkResponseOperationTypes.find((item: DropdownItemProps) => item.key === newTabIndex);
    
if (item) {
    setResponseOperationType(item.value as BulkImportResponseOperationTypes);
    setResponseOperationTypeTab(newTabIndex);
}
```

Reply:
Fixed